res/controllers/Numark-Scratch.midi.xml

```diff
@@ -1,12 +1,102 @@
 <?xml version='1.0' encoding='utf-8'?>
-<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.3.0+">
+<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.5.0+">
   <info>
     <name>Numark Scratch</name>
     <author>Al Hadebe(NotYourAverageAl)</author>
     <description>Mapping for the Numark Scratch Mixer</description>
-    <forums>NumarkScratch</forums>
-    <manual>NumarkScratch</manual>
+    <forums>https://mixxx.discourse.group/t/numark-scratch-mapping/25186</forums>
+    <manual>numark_scratch</manual>
   </info>
+  <settings>
+    <group label="Alternative Mapping">
+      <option
+        variable="invertLoopEncoderFunction"
+        type="boolean"
+        default="false"
+        label="Invert the Loop Encoder functionality">
+        <description>
+          By default the Encoder manages looping and Shift + Encoder scrolls the library/loads track.
+          If this option is selected the Encoder scrolls the library/loads track and Shift + Encoder manages looping.
+        </description>
+      </option>
+    </group>
+    <group label="Mixer Lighting">
+      <option
+        variable="inactiveLightsAlwaysBacklit"
+        type="boolean"
+        default="true"
+        label="Keep LEDs dimmed instead of off when inactive">
+        <description>
+          This will consistently ensure buttons are backlit, which can be helpful in low-light environments.
+          Please note the PAD MODE and CUE buttons are hardware controlled and always dim when inactive.
+        </description>
+      </option>
+    </group>
+    <group label="Beatloop Roll Size">
+      <row orientation="vertical">
+          <option
+              variable="beatLoopRollsSize1"
+              type="enum"
+              label="First Pad">
+              <value label="1/32">0.03125</value>
+              <value label="1/16">0.0625</value>
+              <value label="1/8">0.125</value>
+              <value label="1/4" default="true">0.25</value>
+              <value label="1/2">0.5</value>
+              <value label="1">1</value>
+              <value label="2">2</value>
+              <value label="4">4</value>
+              <value label="8">8</value>
+              <description>The first pad.</description>
```

Comment:
@acolombier I noticed you had Halve & Double beatloop roll sizes if your s4mk3 settings. Have you found this useful in practice?
```
<value label="Halve beatloop size">half</value>
<value label="Double beatloop size">double</value>
```

Reply:
I use it all the time personally! :) 
The size loop encoder is great to adjust loop size widely, but if you want precision of a button, e.g to half the loop for a build up, it is very handy.
I guess having the option is always great to the user?

Reply:
Done

Reply:
>  I guess having the option is always great to the user?

Fyi this has often been a point of debate, on the one hand I fully agree (especially because Mixxx's motto is "dj your way") on the other hand, lots of options may overwhelm the user. So every option needs some thought, not only whether its needed but also what is a sensible and consistent default if its needed.

Reply:
> lots of options may overwhelm the user.

Of course, UX should always be considered, which certainly has been lacking over the past years (these, waveforms, ...).